src/main/java/space/obminyashka/items_exchange/controller/ImageController.java

```diff
@@ -1,19 +1,6 @@
 package space.obminyashka.items_exchange.controller;
 
-import space.obminyashka.items_exchange.dto.ImageDto;
-import space.obminyashka.items_exchange.exception.ElementsNumberExceedException;
-import space.obminyashka.items_exchange.exception.IllegalOperationException;
-import space.obminyashka.items_exchange.exception.UnsupportedMediaTypeException;
-import space.obminyashka.items_exchange.model.Advertisement;
-import space.obminyashka.items_exchange.model.BaseEntity;
-import space.obminyashka.items_exchange.model.User;
-import space.obminyashka.items_exchange.service.AdvertisementService;
-import space.obminyashka.items_exchange.service.ImageService;
-import space.obminyashka.items_exchange.service.UserService;
-import io.swagger.annotations.Api;
```

Comment:
You rewrite the commit history by doing this

Reply:
I did it only because it was removed UserService from here and I had to clean the imports.